core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -861,8 +861,9 @@ object JsonAST {
         case '\r' => "\\r"
         case '\t' => "\\t"
         case c if ((c >= '\u0000' && c < '\u0020')) => "\\u%04x".format(c: Int)
-        case c => c
-      })
+        case c => ""
+      }
+      if (strReplacement.isEmpty) buf.append(c) else buf.append(strReplacement)
```

Comment:
The `isEmpty` branch allows the character to be inserted directly instead of getting converted to a string, which incurs the additional overhead.